packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -161,6 +162,16 @@ const props = defineProps({
     type: String,
     required: true,
   },
+  emptyStateTitle: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  emptyStateDescription: {
+    type: String,
+    required: false,
+    default: '',
+  },
```

Comment:
Update the readme please to reflect the new props

Reply:
Eek. thanks for the heads up.

done deal.
<img width="804" alt="Screen Shot 2023-06-27 at 9 54 27 AM" src="https://github.com/Kong/public-ui-components/assets/103061463/74eadb01-7e51-4d7c-b727-91a14fdb25f2">